recipes/libcxxwrap-julia/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "libcxxwrap-julia" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "62cb4216efb3dc39736630ec747fb35e8685e6cd888b65d6e431905ba45918c6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/JuliaInterop/libcxxwrap-julia/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # Julia has not been packaged for conda on OS X or Windows
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - toolchain
```

Comment:
The compiler syntax has changed. Please update according to the latest docs.

ref: https://conda-forge.org/docs/meta.html#compilers